runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkStreamingPipelineTranslator.java

```diff
@@ -228,4 +238,31 @@ private ReflectiveOneToOneOverrideFactory(
     }
   }
 
+  /**
+   * A {@link PTransformOverrideFactory} that provides overrides for applications of a {@link ParDo}
+   * in the direct runner. Currently overrides applications of <a
```

Comment:
This comment needs to be updated - this overrides exclusively splittable multi-output ParDo's, and only in Flink runner.